airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -602,6 +604,18 @@
   icon: github.svg
   sourceType: api
   releaseStage: generally_available
+  suggestedStreams:
+    streams:
+      - branches
+      - comments
+      - issues
+      - organizations
+      - pull_requests
+      - repositories
+      - stargazers
+      - tags
+      - teams
+      - users
```

Comment:
Here's the new interface for connector developers who only want some streams selected by default

Reply:
I am a bit surprised that this is in the definition yaml file when the schema itself is not present in that file but requires calling a connector function and the suggestedStreams are definitely tied to the schema.

I'm not advocating moving that to a connector function but that might be a good signal that we would benefit from having the schema show up in the definition for API connectors.

Reply:
Yeah... that was the original proposal (https://github.com/airbytehq/airbyte/pull/21437) complete with protocol change.  For posterity, I added a comment in that PR as to why we chose to move this out of the protocol, and you can read more details [here](https://docs.google.com/document/d/1RnoFFr12ZX0WGsjicyC_ErvSib5aP8A0DZ-IPzZcB1g/edit).  

In the near future, all of "actor definitions" will be replaced with "connector metadata", and things will hopefully begin to make more sense.